examples/deepcopy-gen/generators/deepcopy.go

```diff
@@ -762,6 +764,8 @@ func (g *genDeepCopy) doSlice(t *types.Type, sw *generator.SnippetWriter) {
 			sw.Do("}\n", nil)
 		} else if t.Elem.Kind == types.Struct {
 			sw.Do("(*in)[i].DeepCopyInto(&(*out)[i])\n", nil)
+		} else if t.Elem.Kind == types.Alias {
```

Comment:
@thockin I guess you are right. On the other hand we also fall back to `.DeepCopy` one case below. My change does not change much here, only optimized a bit. Struct aliases will work with this. For the full solution we need another case. Or fall back to `g.generateFor(t, sw)` with another `{ in, out := ... }` scope.